### ISSUE TITLE
Wrong indent style false-positive with heredocs

### ISSUE THREAD

**Issue by jasonkarns** [editorconfig-checker/editorconfig-checker#480]
Shell heredocs require leading tabs. editorconfig-checker incorrectly presumes the leading lines to be "indentation" and thus flags `Wrong indent style found (tabs instead of spaces)`.

Sample bats test that triggers the false positive. All lines within the test body are indented with spaces. The Heredoc lines must prefixed with tabs. It is not a configurable choice, it's required syntax and should not be flagged as a violation.

```
@test "file finds multiple files" {
  with_file "$NODENV_ROOT/default-packages" <<<""
  with_file "$HOME/.config/nodenv/default-packages" <<<""
  with_file "$HOME/myconfig/nodenv/default-packages" <<<""
  with_file "$HOME/theirconfig/nodenv/default-packages" <<<""

  XDG_CONFIG_DIRS="$HOME/myconfig:$HOME/theirconfig" run nodenv default-packages files

  assert_success
  assert_output - <<-OUT
	$NODENV_ROOT/default-packages
	$HOME/.config/nodenv/default-packages
	$HOME/myconfig/nodenv/default-packages
	$HOME/theirconfig/nodenv/default-packages
OUT
}
```

editorconfig in effect is:
```
# https://EditorConfig.org
root = true

[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
indent_size = 2
indent_style = space
```

**Comment by proycon** [comment]
I'm having a similar issue with heredocs, our config is:

```
# Unix-style newlines with a newline ending every file
[*.{sh,c}]
end_of_line = lf
insert_final_newline = true
indent_style = tab
#max_line_length = 80
spaces_around_operators = true
charset = utf-8
trim_trailing_whitespace = true
```

And we have a heredoc like: 

```sh
		cat <<- EOF
		Usage: sxmo_brightness.sh [options] [value]

		Options:
		  silent		Prevents notification with (w/x)ob. Use this as first arg to below options.
		  notify		Notify with (w/x)ob current brightness.
		  setvalue VALUE	Set current brightness to VALUE.
		  up VALUE		Up the brightness by 5%% or if VALUE supplied, by VALUE.
		  down VALUE		Lower the brightness by 5%% or if VALUE supplied, by VALUE. Caps out at SXMO_MIN_BRIGHTNESS.
		  getvalue		Get current value.
		EOF
		exit 0
```

Where editorconfig-checker complains about the Options which have an additional two-space indentation, rest is all tabs. I think editorconfig-checker should not just enforce its default indentation policy WITHIN a heredoc.

**Comment by jasonkarns** [comment]
Exact same problem when using a heredoc to assert formatted output. editorconfig-checker is complaining the indent size is wrong (not a multiple).

Heredocs contain all kinds of explicitly-formatted content that can't be changed just to appeace a linter.
